Slambook/index.js

```diff
@@ -10,12 +10,9 @@ document.querySelector('.form').addEventListener('submit', function(event) {
     event.preventDefault(); // Prevent form submission
 
     // Log to console when submit event occurs
-    console.log("I am inside submit");})
-//Write your code above
-
-function showSlamList(){
-//Update the function here
-        let contactName = document.querySelector('input[name="contactName"]').value;
+    console.log("I am inside submit");

```

Comment:
Ensure to add the logic inside the 'showSlamList' function to perform the desired actions.